docs/architecture/adr-020-protobuf-transaction-encoding.md

```diff
@@ -284,27 +285,21 @@ and `FileDescriptor`s and returns a boolean result.
 
 ### Public Key Encoding
 
-Public keys in the Cosmos SDK implement Tendermint's `crypto.PubKey` interface,
-so a natural solution might be to use `Any` as we are doing for other interfaces.
-There are, however, a limited number of public keys in existence and new ones
-aren't created overnight. The proposed solution is to use a `oneof` that:
-
-- attempts to catalog all known key types even if a given app can't use them all
-- has an `Any` member that can be used when a key type isn't present in the `oneof`
+Public keys in the Cosmos SDK implement Tendermint's `crypto.PubKey` interface.
```

Comment:
We could actually keep this sentence:

> so a natural solution might be to use `Any` as we are doing for other interfaces

Or even:

> so the proposed solution is to use `Any` as we are doing for other interfaces

Reply:
you mean we could add another sentence to say that we use `Any` as pubkey field in `BaseAccount`, etc.?

Reply:
yeah, exactly.

Before, we had: "we might use Any, but finally went for oneof". Now, we could just say: "We use Any".